internal/buffer/settings.go

```diff
@@ -117,6 +120,13 @@ func (b *Buffer) DoSetOptionNative(option string, nativeValue interface{}) {
 	if b.OptionCallback != nil {
 		b.OptionCallback(option, nativeValue)
 	}
+
+	err := config.RunPluginFn("onBufferOptionChanged",
```

Comment:
A bit more idiomatic: `if err := config.RunPluginFn(...); err != nil {` ?

Reply:
Not necessarily more readable, but if it is more idiomatic than before...ok.